FrontEndLib/ListBoxWidget.cpp

```diff
@@ -880,29 +890,34 @@ void CListBoxWidget::Paint(
 
 	static const UINT CX_ITEM_INDENT = 2;
 
+#ifdef RUSSIAN_BUILD
+	static const int DY_TEXT = -1; //Because this particular font is a tiny bit too far down.
+#else
+	static const int DY_TEXT = -5; //Because this particular font is always too far down.
+#endif
+
 	//Draw inset area where text appears.
 	SDL_Surface *pDestSurface = GetDestSurface();
 	DrawInset(this->x, this->y, this->w, this->h, this->images[0],
 			pDestSurface, true, false, !IsEnabled());
 
-	if (this->Items.empty())
-		return;
+	const vector<LBOX_ITEM *> items = this->filteredItems;
```

Comment:
Can this also be a reference so we don't copy the vector?  As it's const, I'd think so?